components/channel/Gopkg.toml

```diff
@@ -38,14 +38,24 @@
    branch = "master"
 
 # Match Azure version in common component
-[[override]]
+[[constraint]]
   name = "github.com/Azure/azure-event-hubs-go"
   version = "v1.3.1"
 
-# Align With Knative Eventing
+# Align With Knative Eventing-Contrib
 [[constraint]]
   name = "knative.dev/eventing"
-  version = "v0.13.5"
+  branch = "release-0.14"
+
+# Align With Knative Eventing-Contrib
+[[constraint]]
+  name = "knative.dev/eventing-contrib"
+  branch = "release-0.14"
+
+# Align With Knative Eventing-Contrib
+[[constraint]]
+  name = "github.com/cloudevents/sdk-go"
+  version = "v2.0.0-RC1"
 
```

Comment:
Lets not forget to bump it up to v2 when released.

Reply:
👍